mobile/src/home-view.js

```diff
@@ -244,25 +287,33 @@ class HomeView extends Component {
           </TouchableOpacity>
         )
       }
+
     }
 
+ 
+
     closeConfirm = () => {
-      this.setState({questionAsk: false})
+      this.setState({questionAsk: false, approveVar: false})
     }
 
     originalOrder = (questions) => {
       if (this.state.showRecent === false) {
-        questions.sort(function (a,b){
+        this.dateSort(questions)
```

Comment:
This looks like it is sorting by date, then re-sorting by score immediately afterward.

Reply:
Exactly it should first order by date then score. It should act as a tiebreaker for questions with the same votes?